Entities/MovableObject.cpp

```diff
@@ -744,8 +750,7 @@ bool MovableObject::OnMOHit(HitData &hd)
 }
 
 bool MovableObject::OnMOHit(MovableObject *pOtherMO) {
-    if (pOtherMO != this) { RunScriptedFunctionInAppropriateScripts("OnCollideWithMO", false, false, {pOtherMO, pOtherMO ? pOtherMO->GetRootParent() : nullptr}); }
```

Comment:
Is this OnMOHit used anywhere now? If it is, seems like we still need to run this lua function, even if it's imperfect? If it isn't, why not delete it?

Reply:
I suppose it should be deleted

Reply:
Yeah, burn em all! 🔥